app/priorities/views.py

```diff
@@ -69,7 +70,7 @@ def post_mascot(token, name):
 def get_token():
     form = TokenForm();
     if form.validate_on_submit():
-        token = form.password.data.upper() # FIXME: escape
```

Comment:
The point of this was that we wanted to have case-insensitive tokens, and had all of them uppercased in the backend. Not sure that's what you want :)